PixiEditor/ViewModels/SubViewModels/Main/DiscordViewModel.cs

```diff
@@ -0,0 +1,148 @@
+﻿using DiscordRPC;
+using PixiEditor.Models.DataHolders;
+using PixiEditor.Models.UserPreferences;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PixiEditor.ViewModels.SubViewModels.Main
+{
+    public class DiscordViewModel : SubViewModel<ViewModelMain>
+    {
+        private DiscordRpcClient client;
+        private string clientId;
+        private Document currentDocument;
+
+        public bool Enabled
+        {
+            get => client != null;
+            set
+            {
+                if (Enabled != value)
+                {
+                    if (value)
+                    {
+                        Start();
+                    }
+                    else
+                    {
+                        Stop();
+                    }
```

Comment:
You can use the ternary operator here.

Reply:
How?

Reply:
`value ? Start() : Stop()`

Reply:
<img width="431" alt="nope" src="https://user-images.githubusercontent.com/45312141/106391459-a5809b00-63ed-11eb-9a1d-44ac8061234a.PNG">

Reply:
okay, nvm